### ISSUE TITLE
Slider accessibility

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#14398]
* disabled sliders can bot be focused
* sliders need to announce their new value after it has been changed (see gallery demo)

**Comment by adil-s** [comment]
the slider has still issue on android. when I swipe left and right with talk back, talk back doesn't announce the slider. it just doesn't identify the slider. instead I'm getting a sound feetback that there is something here. but When I explore the screen with finger (dragging the finger up/down and right/left), the talkback focuses on the slider and announces the slider and its value. 

I Tried with this interactive app found at: 
https://api.flutter.dev/flutter/material/Slider-class.html
